modules/ipi-install-ipv6-network-requirements.adoc

```diff
@@ -0,0 +1,71 @@
+// This is included in the following assemblies:
+//
+// ipi-install-prerequisites.adoc
+
+[id="ipi-install-ipv6-network-requirements_{context}"]
+
+= IPv6 Network Requirements
+
+.SLAAC Addressing
+
+If you do not plan to use SLAAC footnote:[Stateless Address AutoConfiguration] addresses on your {product-title} node, then it should be disabled for `baremetal` networks. That means that if the network equipment is configured to send SLAAC addresses when replying to Route Advertisements, you must configure the equipment so that it only sends the route and not the SLAAC address.
```

Comment:
We try to avoid footnotes when possible, so you could just say SLAAC (Stateless Address AutoConfiguration) for the first instance.

Reply:
Fixed.